src/core/mainloop/mainloop.c

```diff
@@ -2719,6 +2742,12 @@ initialize_mainloop_events(void)
 int
 do_main_loop(void)
 {
+  /* For now, starting Tor always counts as user activity. Later, we might
+   * have an option to control this.
+   */
+  reset_user_activity(approx_time());
+  set_network_participation(true);
```

Comment:
reset_user_activity() is different from note_user_activity().  We can't use note_user_activity() here because the events aren't initialized yet (IIRC)

Reply:
Oh my mistake... I thought it was "note_user_activity()"...